### ISSUE TITLE
Choropleth Map with Time Slider

### ISSUE THREAD

**Issue by danielcs88** [artdgn/pages#15]
Impressive maps! If you don't mind me asking, I made an [Choropleth map with unemployment numbers in Florida](https://danielcs88.github.io/html/fl_unemp_map.html) with Plotly. I'd like to create something similar with population numbers over a period of time. I've been looking all over for an example on how to implement that, but to no results.

How would one create a time slider for that?

**Comment by artdgn** [comment]
Hey @danielcs88 sorry for missing your question. 

I've tried playing around with a similar idea (of adding time scale to the map that I'm creating). 
The one example I found that I could successfully use [was this](https://plotly.com/python/plotly-express/#f5560f31-286f-4e4f-8398-2b66fd908995). 

I couldn't really find a way to combine it with the map that I have already (that has buttons for plotting multiple different values), but for plotting a single value on a time range I think it works pretty well.
I only have an uncomitted WIP notebook for playing with that approach.
Hope this helps.

**Comment by danielcs88** [comment]
Hey! Thank you for following up. I actually found a way to graph it!

https://danielcs88.github.io/cues/fl_heatmap_population.html

A useful tip that I found helpful specific for this case is slicing the geojson files for only the values of Florida.

If anybody wants to use my code, here is my version for it:

https://raw.githubusercontent.com/CUES-FIU/fl_population/master/fl_pop_change%20(map).py?token=ADJST7MXAPRIBTO5ELPPIDC64SALO
